index.html

```diff
@@ -0,0 +1,250 @@
+            <li>Ruby on Rails</li>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>ERB</li>
+            <li>RSPec</li>
+          </ul>
+          <a href="https://github.com/Spayco/katchy/" class="card-link link" target="_blank"
+            rel="noreferrer noopener">See project</a>
+        </div>
+      </div>
+
+    </div>
+    <div class="about-me">
+      <section class="about-me-section">
+        <h2>About me</h2>
+        <p class="about-me-text">
+          I am a skilled Full-Stack Web Developer who loves making pages with
+          beautiful UI. I have studied at Microverse, and I am experienced in
+          remote work.
+          <br />
+          I've also been a good helping hand to a lot of developers in
+          different communities!
+          <br />
+          if you'd like to get in contact about potential job opportunities,
+          please reach out!
+        </p>
+        <a href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+          class="link" target="_blank" rel="noreferrer noopener">
+          Get my resume
+        </a>
+        <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+      </section>
+      <hr />
+      <div class="about-me-section stacks">
+        <ul>
+          <li class="languages list-title">
+            Languages
+            <ul>
+              <li>JavaScript</li>
+              <li>TypeScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Sass</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="frameworks list-title">
+            Frameworks
+            <ul>
+              <li>React & Redux</li>
+              <li>Material-UI</li>
+              <li>Jest</li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="skills list-title">
+            Skills
+            <ul>
+              <li>GitHub</li>
+              <li>CodePen</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+              <li>WSL</li>
+              <li>Remote Pair-Programming</li>
+              <li>Teamwork</li>
+              <li>Mentoring</li>
+              <li>Code Reviews</li>
+              <li>Mobile/Responsive Development</li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <section class="contact">
+      <div class="call-to-action">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in</h2>
+      </div>
+      <div class="contact-form">
+        <form class="form" action="https://formspree.io/f/mpzbwzaj" method="POST">
+          <div class="form-group">
+            <input type="text" class="form-control" id="name" placeholder="Full name" name="name" required/>
```

Comment:
- [ ] Please, make sure the name field has a max length of 30 chars by adding `maxlength="30"` attribute as mentioned in  [the requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)
---------
![curriculum-html-css_3_contact_form md at main · microverseinc_curriculum-html-css (1)](https://user-images.githubusercontent.com/71507900/147368848-62decf3f-2a7c-4437-800b-2a09c7551d2c.png)